frontend/src/views/main/project/TestSuiteNew.vue

```diff
@@ -72,10 +90,34 @@ let suite = ref<TestSuiteNewDTO | null>(null);
 let registry = ref<TestCatalogDTO | null>(null);
 let tab = ref<any>(null);
 let selectedTest = ref<SuiteTestDTO | null>(null);
-let inputs = ref<{}>([]);
+let inputs = ref<{
+  [name: string]: string
+}>({});
+const allDatasets = ref<{ [key: string]: DatasetDTO }>({});
+const allModels = ref<{ [key: string]: ModelDTO }>({});
+
 onMounted(async () => {
-  inputs.value = await api.getTestSuiteNewInputs(props.projectId, props.suiteId);
-  suite.value = await api.getTestSuiteNew(props.projectId, props.suiteId);
-  registry.value = await api.getTestsCatalog(props.projectId);
+  // Call api in parallel to shorten loading time
+  const [
+    inputResults,
+    suiteResults,
+    registryResult,
+    datasets,
+    models
+  ] = await Promise.all([
+    api.getTestSuiteNewInputs(props.projectId, props.suiteId),
+    api.getTestSuiteNew(props.projectId, props.suiteId),
+    api.getTestsCatalog(props.projectId),
+    api.getProjectDatasets(props.projectId),
+    api.getProjectModels(props.projectId)
+  ]);
```

Comment:
Not so much a review comment as a thought for later, but when the Pinia migration gets merged, we should probably look into storing a lot of project information into the store itself. We'd probably make the UI a bit faster that way